### ISSUE TITLE
I was working with the VGGish model train_demo python file and could not understand the last part of function - _get_examples_batch

### ISSUE THREAD

**Issue by tusharpoddar** [tensorflow/models#8754]
I was working with the VGGish model train_demo python file. I could not understand why we are returning all the labels and spectrograms each time for training in the function _get_examples_batch. The question I had was since we have batch number shouldn't we supply a different subset of the data each time we train the same model?

**Comment by ravikyram** [comment]
@tusharpoddar 

Please, fill [issue template.](https://github.com/tensorflow/models/issues/new/choose).

Please, let us know which pretrained model you are using and share related code .Thanks!

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
